recipes/pypolyagamma-3/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pypolyagamma-3" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
+
+build:
+  number: 0
+  # We lack gsl on windows
+  skip: True  # [win]
+  skip: True  # [not py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc  # [osx]
+    - gsl
+    - numpy
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  commands:
+    - python -c "import pypolyagamma;"
+
+about:
+  home: https://github.com/Savvysherpa/pypolyagamma-3
```

Comment:
>  If it goes against the purpose of conda forge or violates some policy we can try to work something else out.

It is not against our policies per-se, but it would be nice to work as standard OSS projects by packaging the original source and adding your changes as a patch while you wait for them to be merge. Does that makes sense to you?